src/main/java/se/liu/ida/hefquin/engine/federation/GraphQLEndpoint.java

```diff
@@ -58,4 +58,9 @@ public interface GraphQLEndpoint extends FederationMember {
      * If the parameter names provided doesn't correspond to an object type or a GraphQLEntrypointType for the endpoint return null.
      */
     public GraphQLEntrypoint getEntrypoint(final String objectTypeName, final GraphQLEntrypointType fieldType);
+
+    /*
+     * Uses System.out to print information about the endpoint such as GraphQL object types and their fields.
+     */
+    public void printInformation();
```

Comment:
Is this method really necessary? Usually this kinds of systems do not use the `System.out.print` methods to log information. If you want this info for debugging or to generate files with, I would recommend to implement the `toString()` method instead.